README.md

```diff
@@ -25,6 +25,8 @@ Cloud](https://storage.googleapis.com/cloudrun/button.svg)](https://console.clou
 1. Make sure the repository has a Dockerfile, so it can be built using the
    `docker build` command.
 
+1. *Optional* To use a different git branch, add a `cloudshell_git_branch=BRANCH_NAME` query parameter
```

Comment:
I think this deserves a new section, like `Customizing source repository` where you can talk about both cloudshell_git_branch, as well as cloudshell_dir (also undocumented).

Since most people won't need to customize these two, I'm inclined to have them outside the Instructions section relevant to ~every user.

Reply:
New section added